### ISSUE TITLE
-2147483 wet morale penalty

### ISSUE THREAD

**Issue by l29ah** [CleverRaven/Cataclysm-DDA#31855]
# Describe the bug

I was just walking outside during a snowy weather.

# Versions and configuration

0.D-4963-g3eec9d8b2a-dirty

# Additional context

[2019.06.04.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/3327507/2019.06.04.zip)

**Comment by jbytheway** [comment]
I looked at your save, and I agree it's become broken.  But new wetness morale penalties seem fine.  I have no good ideas of how to debug this.

This is the second issue I've looked into which seems to be caused by "impossible" values.  I'd like to say we might have some out-of-bounds write, but if that were so it ought to be causing crashes too.

Probably have to wait and see whether anyone else gets a similar issue.
